### DISCUSSION TITLE
请问如何通过代码或者命令复制文件到剪切板

### DISCUSSION THREAD

**Discussion by wenfer** [linuxdeepin/developer-center#5144] [Development Questions | 开发答疑] [answered]
我看ctrl+c是可以复制文件的，但是如果使用xsel命令就只能复制文本，请问，如果能够引用ctrl+c的功能把文件复制到剪切板呢

**Comment by Decodetalkers** [comment]
xclip ,wl-copy

**Comment by BLumia** [comment, marked as answer]
复制内容的属性实质是剪切版里存储的 MimeType 的差异，比如复制文件就可以是 `text/uri-list`。具体的话 xclip 之类就可以满足需求，可以参考 https://askubuntu.com/questions/211250/how-to-copy-many-files-to-clipboard-from-the-command-line
